to.us.scrollprotocol.Profectus.yml

```
@@ -0,0 +1,49 @@
+id: to.us.scrollprotocol.Profectus
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: profectus
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-download:rw
+  # Audio
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+modules:
+  - name: sdl2-gfx
+    buildsystem: autotools
```

SDL2_gfx build system is broken:

```suggestion
    buildsystem: autotools
    build-options:
        arch:
            aarch64:
                config-opts:
                    - --disable-mmx
```

---

Ok, I've just added this. Does the queued test use the latest commit or the commit at the time the queue happened? Do I need to reinvoke the buildbot after this change?

---

Also, sorry. Didn't realize I could commit suggested changes directly like that!